calcom/cal.com
# Salesforce app

## What does this PR do?

Salesforce app to add new contacts and events from bookings. 

Also, this PR:
 - Renames HubSpot and Close.com apps to clean up URLs, which is why there is an additional task to take care of those renamed apps
 - Introduces a way to surfice errors and warnings from installed apps runs into the Apps Status in the organizer email. And to avoid surfacing errors that could not be as descriptive or updated as we want, only errors within a `calError` property will be surfaced. Warnings are totally new and also require a `calWarning` property to be  defined.

### Pending tasks 
 - [ ] Go into HubSpot Developer Account holding the Cal.com app and modify part of the Redirect URI in it from "hubspotothercalendar" to "hubspot"
 - [x] Create Salesforce app for Staging and for Production copying the one created for Dev, credentials for team@cal.com in Bitwarden
 - [x] Add instructions to the README to create a Salesforce app for this Cal app to work

Loom Video: https://www.loom.com/share/f0d65ef09a5b474d85a7c62343bf9b9f

## Type of change


## How should this be tested?

Install Salesforce app, receive a booking, and see the attendee(s) be created in Salesforce as contacts with an event representing the booking.
